test/puppetlabs/ssl_utils/core_test.clj

```diff
@@ -660,7 +660,7 @@
       (testing (str "should be able to load cert chain from reader in "
                     "keystore and retrieve it back")
         (let [private-key-file (open-ssl-file "private_keys/localhost.pem")
-              cert-file        (open-ssl-file "certs/multiple.pem")
+              cert-file        (open-ssl-file "certs/localhost.pem")
```

Comment:
I don't see how this is related?

Reply:
private_keys/localhost.pem has one key in it (single key in a chain) and certs/multiple.pem has two (an intermediate chain). When you run with this in Bouncy Castle FIPS, you get a "RSA keys do not have the same modulus" exception. The BC provider is more strict than the Java one.

This chooses the cert-file that matches the private key.